### PR TITLE
add s3 serving documents

### PR DESCRIPTION
**Pull request by LeoQuote**
related to: https://github.com/pypa/bandersnatch/issues/552

### CODE REVIEW

**Comment by LeoQuote** [comment]
maybe skip news? 

I just added a very simple instruction, mostly just a link to s3 document and consulting cloud provider....Hope this could be helpful.

I don't have an AWS account either, In fact I was setting up mirror on Tencent cloud, but I believe it is feasible in AWS as well.

File: docs/storage_options.md [hunk 1]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
 S3 Bandersnatch mirrors are designed to be served with s3 static sites and
 can also be used with the Amazon CDN service or another CDN service.
 
+I assume you have already set up an AWS account and S3 bucket, and the Bandersnatch sync job has been proceeded.
```
Comment (cooperlees) [inline docs/storage_options.md:65]:
```suggestion
I assume you have already set up an AWS account and S3 bucket, and the Bandersnatch sync job has successfully ran.
```

File: docs/storage_options.md [hunk 2]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
...
+
+### Enabling website hosting for the bucket
+When you enable the website hosting for a bucket, this bucket can be viewed as static website. Using the s3 domain or your customized domain.
```
Comment (cooperlees) [inline docs/storage_options.md:68]: NIT - Markdown wants a \n between all headings - Please do to others.

- I need to add prettier auto formattting to our .md files ...

```suggestion
### Enabling website hosting for the bucket

When you enable the website hosting for a bucket, this bucket can be viewed as static website. Using the s3 domain or your customized domain.
```

File: docs/storage_options.md [hunk 3]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
...
+
+Please read Amazon documents to get detailed instructions: https://docs.aws.amazon.com/AmazonS3/latest/userguide/EnableWebsiteHosting.html
```
Comment (cooperlees) [inline docs/storage_options.md:70]: We could make this links ...
```suggestion
Please read Amazon documents to get [detailed instructions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/EnableWebsiteHosting.html)
```

File: docs/storage_options.md [hunk 4]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
...
+
+Most cloud provider who provide s3-compatible service will provide this service as well. Please consult to your service assistant to get detailed instructions.
```
Comment (cooperlees) [inline docs/storage_options.md:72]:
```suggestion
Most cloud provider who provide a s3-compatible service will provide this service as well. Please consult to your service assistant to get detailed instructions.
```

File: docs/storage_options.md [hunk 5]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
...
+
+### Use CloudFront or other cdn service to speed up the static mirror(optional)
+If your mirror is targeted to global clients, you can use CloudFront or other CDN service to speed up the mirror.
+
+Please read Amazon documents to get detailed instructions: https://docs.aws.amazon.com/AmazonS3/latest/userguide/website-hosting-cloudfront-walkthrough.html
```
Comment (cooperlees) [inline docs/storage_options.md:77]:
```suggestion
Please read Amazon documents to get [detailed instructions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/website-hosting-cloudfront-walkthrough.html)
```

File: docs/storage_options.md [hunk 6]
```diff
@@ -60,9 +62,37 @@ signature_version = s3v4
...
+But there are two main disadvantages:
+1. The url is quite long and exposing the structure of bucket.
+2. Users will be able to view all content in the bucket, including bandersnatch todo file and status file.
+
+It is strongly recommended to set redirect or url rewrite for CDN. Please contact your service assistant for detailed instructions.
+
 ## SwiftStack Storage
 
-To enable SwiftStack support the optional `ssift` install must be done:
+To enable SwiftStack support the optional `swift` install must be done:
```
Comment (cooperlees) [inline docs/storage_options.md:95]: Thanks! haha

**Review by cooperlees — CHANGES_REQUESTED** [review]
This is great. Lets just get a few fixes and I'll merge away. Appreciate this!

**Review by cooperlees — APPROVED** [review]

**Pull request merged**